app/write_square_test.go

```diff
@@ -54,12 +54,12 @@ func Test_finalizeLayout(t *testing.T) {
 		},
 		{
 			squareSize:      4,
-			nonreserveStart: 6,
+			nonreserveStart: 7,
 			ptxs: generateParsedTxsWithNIDs(
```

Comment:
[for reviewers] this test now generates 7 compact shares so the `nonreserveStart` field had to be updated and one blob had to be removed to fit in a square size of 4